<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1089</width>
    <height>795</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLCDNumber" name="hourLCD">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>161</width>
      <height>141</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="value" stdset="0">
     <double>88.000000000000000</double>
    </property>
   </widget>
   <widget class="QLCDNumber" name="minuteLCD">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>80</y>
      <width>161</width>
      <height>141</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="value" stdset="0">
     <double>88.000000000000000</double>
    </property>
   </widget>
   <widget class="QLCDNumber" name="secondLCD">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>80</y>
      <width>161</width>
      <height>141</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="value" stdset="0">
     <double>88.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="greetingLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>501</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>greeting label</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="timeZoneBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>-Change Time Zone-</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Seattle (PDT)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Seoul (KST)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>New York (EDT)</string>
     </property>
    </item>
   </widget>
   <widget class="QLCDNumber" name="secondLCD_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>260</y>
      <width>131</width>
      <height>121</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="value" stdset="0">
     <double>88.000000000000000</double>
    </property>
   </widget>
   <widget class="QLCDNumber" name="hourLCD_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>141</width>
      <height>121</height>
     </rect>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="value" stdset="0">
     <double>88.000000000000000</double>
    </property>
   </widget>
   <widget class="QLCDNumber" name="minuteLCD_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>260</y>
      <width>141</width>
      <height>121</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="value" stdset="0">
     <double>88.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="imageDownloadButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Set ZIP Code</string>
    </property>
   </widget>
   <widget class="QLabel" name="imageLabel">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>70</y>
      <width>441</width>
      <height>311</height>
     </rect>
    </property>
    <property name="text">
     <string>imageLabel</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="zipCodeEdit">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>20</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>98119</string>
    </property>
   </widget>
   <widget class="QPushButton" name="weatherDownloadButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>390</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Weather Download</string>
    </property>
   </widget>
   <widget class="QLabel" name="weatherLabel">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>440</y>
      <width>441</width>
      <height>311</height>
     </rect>
    </property>
    <property name="text">
     <string>imageLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1089</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
